src/Config/InfectionConfig.php

```diff
@@ -243,4 +243,9 @@ function ($excludeDir) use ($srcDir) {
 
         return $excludedDirs;
     }
+
+    public function getTempDir(): string
```

Comment:
If we are not going to use `getTemporaryDirectory` I would rather go for `getTmpDir` which IMO is more standard. I does make sense with `sys_get_temp_dir()` but then PHP also has `upload_tmp_dir()` and I see `tmp` way more used in Unix, Java, Python, Ruby etc.